source/chapters/user_interface.rst

```diff
@@ -666,15 +676,28 @@ Equalizers and Gain Knobs
    EQ Controls
 
 **Gain Knob**
-  Above these knobs, the gain knob allows you to adjust the gain of the deck. In
-  general, you should adjust this knob so that at full-volume the deck's audio
-  is just at the peak of the center VU meters. This is so you can achieve the
-  widest dynamic range for your track.
+  Above these knobs, the gain knob allows you to adjust the gain of the deck. 
+  Use this to compensate for the differences in recording levels between tracks.
+  In general, you should adjust this knob so that at full-volume the deck's
+  audio is just at the peak of the center VU meters. This is so you can achieve
+  the widest dynamic range for your track.
+  
+  .. note:: By default, Mixxx automatically adjusts the gain of tracks to sound
+            like they are at approximately the same volume. Your tracks must be
+            :ref:`analyzed<library-analyze>` in advance to take advantage of
```

Comment:
Sorry, now my comment confuses me as well. Here some new thoughts: 
Use Loudness instead of volume. 
Mixxx does not adjust THE gain knob, it adjusts an additional hidden gain.
If the track already has a replaygain info in its metadata, this will be used.  
Not sure if this all matters here.  

Draft, please impove!): 

By default, Mixxx automatically applies an additional ReplayGain for an equal loudness at unity gain.  
Missing ReplayGain values are calculated during track load or in advandes by :ref:`analyzed<library-analyze>`